packages/integration-sdk-testing/src/createStepTest.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Entity,
+  IntegrationInstanceConfig,
+  IntegrationInvocationConfig,
+  Relationship,
+} from '@jupiterone/integration-sdk-core';
+import { StepTestConfig } from './config';
+import { executeStepWithDependencies } from './executeStepWithDependencies';
+import { setupRecording, SetupRecordingInput } from './recording';
+
+export type WithRecordingParams = SetupRecordingInput;
+
+export async function withRecording(
+  withRecordingParams: WithRecordingParams,
+  cb: () => Promise<void>,
+) {
+  const recording = setupRecording(withRecordingParams);
+
+  try {
+    await cb();
+  } finally {
+    await recording.stop();
+  }
+}
+
+type AfterStepCollectionExecutionParams = {
+  stepConfig: StepTestConfig<
+    IntegrationInvocationConfig<IntegrationInstanceConfig>,
+    IntegrationInstanceConfig
```

Comment:
I often see us use things like `TConfig` or `TInvocationConfig`. I couldn't tell if those were necessary here or not.